c7n/resources/rdscluster.py

```diff
@@ -56,6 +69,7 @@ class resource_type(TypeInfo):
         arn_separator = ":"
         enum_spec = ('describe_db_clusters', 'DBClusters', None)
         name = id = 'DBClusterIdentifier'
+        filter_name = "Filters"
```

Comment:
yeah we can skip the metadata here, its pretty custom per se,  the metadata is for cases where the query machinery can automatically pass through.

Reply:
ok, thx